### PR TITLE
Access error: Auth name cannot be updated for SSO users

### PR DESCRIPTION
**Pull request by stefanogramm**
adding  `Access API error: auth_domain_cannot_be_updated_dash_sso`
from https://github.com/stefanogramm/cloudflare-docs/blob/production/content/cloudflare-one/faq/teams-troubleshooting.md

to production: `cloudflare-docs/blob/production/content/cloudflare-one/faq/teams-troubleshooting.md`

### CODE REVIEW

**Comment by ranbel** [comment]
@stefanogramm To clarify, users see the message `Access API Error: auth_domain_cannot_be_updated_dash_sso` when changing their team domain on the dashboard?

File: content/cloudflare-one/faq/teams-troubleshooting.md [hunk 1]
```diff
@@ -139,3 +139,7 @@ Cloudflare enforces a 270-second idle timeout on TCP connections that go through
 
 If `cloudflared` returns error `error="remote error: tls: handshake failure"`, check to make sure the hostname in question is covered by a SSL certificate. If using a multi-level subdomain, an advanced certificate may be required as the Universal SSL will not cover more than one level of subdomain. This may surface in the browser as `ERR_SSL_VERSION_OR_CIPHER_MISMATCH`.
 
+## Access API error auth_domain_cannot_be_updated_dash_sso.
```
Comment (ranbel) [inline content/cloudflare-one/faq/teams-troubleshooting.md]:
```suggestion
## I see `Access api error auth_domain_cannot_be_updated_dash_sso`.
```

File: content/cloudflare-one/faq/teams-troubleshooting.md [hunk 2]
```diff
@@ -139,3 +139,7 @@ Cloudflare enforces a 270-second idle timeout on TCP connections that go through
...
+
+You may see this error while trying to change your [ZT auth domain](https://developers.cloudflare.com/cloudflare-one/faq/teams-getting-started-faq/#whats-a-team-domainteam-name) aka "Team domain" and SSO is enabled for your account.
+Currently customers that have [SSO](https://developers.cloudflare.com/cloudflare-one/applications/configure-apps/dash-sso-apps/) enabled, cannot change their ZT auth domain, contact support for more details.
```
Comment (ranbel) [inline content/cloudflare-one/faq/teams-troubleshooting.md]:
```suggestion
Cloudflare Dashboard SSO does not currently support team domain changes. Contact support for more details.
```

File: content/cloudflare-one/faq/teams-troubleshooting.md [hunk 3]
```diff
@@ -139,3 +139,7 @@ Cloudflare enforces a 270-second idle timeout on TCP connections that go through
 
 If `cloudflared` returns error `error="remote error: tls: handshake failure"`, check to make sure the hostname in question is covered by a SSL certificate. If using a multi-level subdomain, an advanced certificate may be required as the Universal SSL will not cover more than one level of subdomain. This may surface in the browser as `ERR_SSL_VERSION_OR_CIPHER_MISMATCH`.
 
+## Access API error auth_domain_cannot_be_updated_dash_sso.
+
+You may see this error while trying to change your [ZT auth domain](https://developers.cloudflare.com/cloudflare-one/faq/teams-getting-started-faq/#whats-a-team-domainteam-name) aka "Team domain" and SSO is enabled for your account.
```
Comment (ranbel) [inline content/cloudflare-one/faq/teams-troubleshooting.md]:
```suggestion
This error appears if you try to change your [team domain](/cloudflare-one/faq/teams-getting-started-faq/#whats-a-team-domainteam-name) while the [Cloudflare Dashboard SSO](/cloudflare-one/applications/configure-apps/dash-sso-apps/) feature is enabled on your account.
```

**Review by ranbel — APPROVED** [review]

**Comment by stefanogramm** [comment]
> @stefanogramm To clarify, users see the message `Access API Error: auth_domain_cannot_be_updated_dash_sso` when changing their team domain on the dashboard?

Yup! that's correct, users with SSO enabled trying to update [Zero Trust team name](https://developers.cloudflare.com/cloudflare-one/glossary/#team-name), will face this error on the dashboard.

**Pull request merged**